optuna/visualization/_pareto_front.py

```diff
@@ -1,10 +1,10 @@
 from __future__ import annotations
 
 import collections
+from collections.abc import Callable
+from collections.abc import Sequence
```

Comment:
I guess we can remove `import collections` by replacing `collections.abc.Sequence` with just `Sequence` below:

https://github.com/optuna/optuna/blob/98578c42dfc3fb60959f0bfdf8d208d659ab1b6f/optuna/visualization/_pareto_front.py#L305